### DISCUSSION TITLE
script.outbound-links.js 404 with Cloudfront behaviour

### DISCUSSION THREAD

**Discussion by espen** [plausible/analytics#1201] [Anything Else]
Path `/js/script.js` works but `/js/script.outbound-links.js` results in a 404. Anyone got outbound links script working via Cloudfront?

**Comment by Vigasaurus** [comment]
You need to use `plausible.outbound-links.js` - `script.js` is just an alias for `plausible.js` and doesn't have the variants.

**Reply by espen** [reply]
Ok, I thought it was using `script.js` to avoid being blocked. Also the docs says to use `script.outbound-links.js` : https://plausible.io/docs/proxy/guides/cloudfront

**Reply by Vigasaurus** [reply]
Ah, yeah I misunderstood - with CloudFront you'd need to configure the different variants as well, i.e. add an additional path pattern, but it would indeed need to be plausible.type.js still, as the regular plausible service doesn't serve `script.outbound-links.js` at all. As far as I'm aware the docs are slightly incorrect on that matter.

**Comment by espen** [comment]
Created issue https://github.com/plausible/analytics/issues/1204
